scripts/globals/spells/bluemagic/pinecone_bomb.lua

```diff
@@ -0,0 +1,65 @@
+-----------------------------------------
+-- Spell: Pinecone Bomb
+-- Additional effect: Sleep. Duration of effect varies with TP
+-- Spell cost: 48 MP
+-- Monster Type: Plantoids
+-- Spell Type: Physical (Piercing)
+-- Blue Magic Points: 2
+-- Stat Bonus: STR+1
+-- Level: 36
+-- Casting Time: 2.5 seconds
+-- Recast Time: 26.5 seconds
+-- Skillchain Element(s): Fire (can open Scission or Fusion and can close Liquefaction)
+-- Combos: None
+-----------------------------------------
+require("scripts/globals/bluemagic")
+require("scripts/globals/status")
+require("scripts/globals/magic")
+-----------------------------------------
+
+function onMagicCastingCheck(caster,target,spell)
+    return 0
+end
+
+function onSpellCast(caster,target,spell)
+    local params = {}
+    -- This data should match information on http://wiki.ffxiclopedia.org/wiki/Calculating_Blue_Magic_Damage
+    params.tpmod = TPMOD_CRITICAL
+    params.dmgtype = DMGTYPE_PIERCE
+    params.scattr = SC_LIQUEFACTION
+    params.numhits = 1
+    params.multiplier = 1.25
+    params.tp150 = 1.25
+    params.tp300 = 1.25
```

Comment:
I believe these values are stale from when I branched this code from another spell.

It should be `2.25`, I believe. FFXIclopedia article also agrees. It has it listed as `multiplier` for pinecone bomb: https://ffxiclopedia.fandom.com/wiki/Calculating_Blue_Magic_Damage

Reply:
Also `params.azuretp` does not seem to be used anywhere. From what I can tell. But I'll just set it as `2.25` as well since the damage multiplier seems to be linear.

Reply:
Added a commit to address this comment.

Reply:
The commit was on the branch for your PR to DSP 😉 

I'll go ahead and mark this comment as resolved though, assuming a similar commit will make it to this PR.

Reply:
I added a commit to both branches. I could have sworn I pushed it, but I must have forgot to `git push` or had a typo and didn't notice it failed.

Should be there now.